enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -69,35 +71,35 @@ Write tooling that automatically:
 
 The following statement (or a link to this section) can be pasted into bugs when adding `ImpactStatementRequested`:
 
-We're asking the following questions to evaluate whether or not this bug warrants blocking an upgrade edge from either the previous X.Y or X.Y.Z.
+We're asking the following questions to evaluate whether or not this bug warrants changing update recommendations from either the previous X.Y or X.Y.Z.
 The ultimate goal is to avoid delivering an update which introduces new risk or reduces cluster functionality in any way.
 Sample answers are provided to give more context and the ImpactStatementRequested label has been added to this bug.
 When responding, please remove ImpactStatementRequested and set the ImpactStatementProposed label.
 The expectation is that the assignee answers these questions.
 
-Who is impacted?  If we have to block upgrade edges based on this issue, which edges would need blocking?
-* example: Customers upgrading from 4.y.Z to 4.y+1.z running on GCP with thousands of namespaces, approximately 5% of the subscribed fleet
-* example: All customers upgrading from 4.y.z to 4.y+1.z fail approximately 10% of the time
+Who is impacted?  Which 4.y.z to 4.y'.z' updates?  Which types of clusters?  How does a cluster admin determine they are vulnerable?
+* example: Customers upgrading from 4.y.Z to 4.y+1.z running on GCP with thousands of namespaces, approximately 5% of the subscribed fleet.  Check your vulnerability with `oc ...` or the following PromQL `count (...) > 0`.
+* example: All customers upgrading from 4.y.z to 4.y+1.z fail approximately 10% of the time.  Check your vulnerability with `oc adm upgrade` to show your current cluster version.
 
-What is the impact?  Is it serious enough to warrant blocking edges?
-* example: Up to 2 minute disruption in edge routing
-* example: Up to 90 seconds of API downtime
-* example: etcd loses quorum and you have to restore from backup
+What is the impact?  Is it serious enough to warrant removing update recommendations?  How does a cluster admin determine they are getting bitten?
```

Comment:
`How does a cluster admin determine they are vulnerable?` and `How does a cluster admin determine they are getting bitten?` is almost the same thing IMO.

Reply:
the distinction is that "vulnerable" would go into conditional update preconditions.  E.g. "if I have `$CONFIG`, and I update to `$VERSION`, I'll be at risk".  Like [high control plane memory and 4.9.<28 makes you vulnerable to etcd data inconsistency][1].  "Getting bitten" is "the bad thing is currently happening to you" and goes into the recovery KCS.  Like "[`etcdctl endpoint status -w fields --cluster` shows persistent, divergent revisions][1]".  We can protect folks from vulnerabilities without waiting for them to get bitten.  Some folks who we fail to protect from vulnerabilities will _still_ not get bitten.

[1]: https://access.redhat.com/solutions/6849521

Reply:
Yup.  I've pushed 92525969 -> 18de1352 making all of this more explicit.